python/tutorials/10-experimental-block-pointer.py

```diff
@@ -373,7 +372,7 @@ def matmul(a, b, accum_dtype, res_dtype):
         # Note: the torch.matmul and Triton implementations uses different
         # algorithms so we need to adjust tolerance.
         rtol = 1e-2 if dtype == torch.bfloat16 or accum_dtype in [torch.float16, torch.bfloat16] else 1e-3
-        atol = 1e-2 if accum_dtype == torch.bfloat16 else 1e-3 if accum_dtype == torch.float16 else 1e-4
+        atol = 1e-1 if dtype == torch.float32 else 1e-2 if accum_dtype == torch.bfloat16 else 1e-3 if accum_dtype == torch.float16 else 1e-4
```

Comment:
```suggestion
        atol = 1e-1 if accum_dtype == torch.float32 else 1e-2 if accum_dtype == torch.bfloat16 else 1e-3 if accum_dtype == torch.float16 else 1e-4
```
saying that, float32 having 2 orders of magnitude more absolute error seems very weird to me.

Reply:
3 orders. I can expect that depending on the stability of a system. However I don't have analysis on this one. Relative error is still `1e-4`.

Reply:
This suggestion is probably wrong since there is only one instance where dtype is float32